devel/python/python/ert/enkf/local_config.py

```diff
@@ -14,36 +14,95 @@
 #  See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html> 
 #  for more details.
 from ert.cwrap import BaseCClass, CWrapper
-from ert.enkf import ENKF_LIB
-
-from ert.util import StringList
+from ert.enkf import ENKF_LIB, LocalUpdateStep
+from ert.enkf.local_ministep import LocalMinistep
 
 
 class LocalConfig(BaseCClass):
+    
     def __init__(self):
         raise NotImplementedError("Class can not be instantiated directly!")
+        
+    def free(self):
+        LocalConfig.cNamespace().free(self)
 
-    def get_config_files(self):
+    def getConfigFiles(self):
         """ @rtype: StringList """
         return LocalConfig.cNamespace().get_config_files(self).setParent(self)
 
-    def clear_config_files(self):
+    def clearConfigFiles(self):
         LocalConfig.cNamespace().clear_config_files(self)
 
-    def add_config_file(self, filename):
+    def addConfigFile(self, filename):
+        assert isinstance(filename, str)
         LocalConfig.cNamespace().add_config_file(self, filename)
-
-    def free(self):
-        LocalConfig.cNamespace().free(self)
-
+        
+    def writeLocalConfigFile(self, filename):
+        assert isinstance(filename, str)
+        LocalConfig.cNamespace().write_local_config_file(self, filename)
+           
+    def createUpdatestep(self, update_step_key):
```

Comment:
I guess it would be good to have `getUpdateStep(), getMinistep(), getObsdata()` and `getDataset()`methods.